server/tilesource/base.py

```diff
@@ -1349,6 +1370,16 @@ def tileIterator(self, format=(TILE_FORMAT_PIL, ), resample=False,
                 only the stride of the tiles.  This is a dictionary containing:
             x: the overlap on the left and right in pixels.
             y: the overlap on the top and bottom in pixels.
+            edges: if True, then the edge tiles will exclude the overlap
+                distance.  If unset or False, the edge tiles are full size.
+                    As an example, suppose an image that is 8 pixels across
+                (01234567) and a tile size of 5 is requested with an overlap of
+                2.  If the edges option is False (the default), the following
+                tiles are returned: 01234, 12345, 23456, 34567.  Each tile
```

Comment:
you can still think of it as per-side. For example for a 1x10 image (0123456789) with a tile size of 4 and  odd overlap of 1 ... we would get (0123, 3456, 6789) or (0, 0123, 3456, 6789, 9) depending on how you want to handle edges. In both of these results, the overlap of each tile with the tile on left or right is equal to 1. 

This is how most computer vision or image analysis